core/client/fs/src/main/java/alluxio/client/file/cache/store/LocalPageStoreDir.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.file.cache.store;
+
+import static alluxio.client.file.cache.store.PageStoreDir.getFileBucket;
+
+import alluxio.client.file.cache.PageId;
+import alluxio.client.file.cache.PageInfo;
+import alluxio.client.file.cache.PageStore;
+import alluxio.client.file.cache.evictor.CacheEvictor;
+
+import com.google.common.base.Preconditions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.function.Consumer;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import javax.annotation.Nullable;
+
+/**
+ *
+ */
+public class LocalPageStoreDir extends QuotaManagedPageStoreDir {
+  private static final Logger LOG = LoggerFactory.getLogger(LocalPageStoreDir.class);
+
+  private final LocalPageStoreOptions mPageStoreOptions;
+  private final int mFileBuckets;
+  private final Pattern mPagePattern;
+
+  private PageStore mPageStore;
```

Comment:
should be `LocalPageStore` instead?

Reply:
Yep, using LocalPageStore is better, changed it to LocalPageStore, thanks!